livestream/views.py

```diff
@@ -0,0 +1,20 @@
+"""Livestream Views"""
+from rest_framework import status
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.response import Response
+
+from django.conf import settings
+
+from livestream.api import get_upcoming_events
+from open_discussions.permissions import AnonymousAccessReadonlyPermission
+
+
+@api_view()
+@permission_classes([AnonymousAccessReadonlyPermission])
```

Comment:
If we're only using `GET` for this API and everyone is allowed to read from it then we can just pass an empty list here and pass `["GET"]` as an argument to `api_view`

Reply:
cool, will do